src/main/webapp/app/exercises/modeling/manage/modeling-exercise.component.html

```diff
@@ -1,8 +1,9 @@
 <div>
     <h4 class="d-flex mb-3 flex-wrap justify-content-end">
         <span class="mb-1 me-auto">
-            <span *ngIf="course && !showHeading">{{ course.title }} - </span
-            ><span *ngIf="modelingExercises && showHeading">{{ getAmountOfExercisesString(modelingExercises) }} </span>
+            <span *ngIf="course && !showHeading">{{ course.title }} - </span>
+            <span *ngIf="modelingExercises && showHeading && !filter.isEmpty()">{{ getAmountOfExercisesString(filteredModelingExercises) }} / </span>
+            <span *ngIf="modelingExercises && showHeading">{{ getAmountOfExercisesString(modelingExercises) }} </span>
```

Comment:
You didn't address this. Please either do the updates or add a comment.

Reply:
Since this is how it was done before and would add the same code to each of the exercise.compontent.ts files, I would leave it as it is.
I think this could be addressed in a future PR that takes care of the code duplication in all of these files.